admin-ui/src/pages/settings-page.tsx

```
@@ -2036,12 +2120,15 @@ function SettingsPageView({
           >
             <AdvancedSettingsCard
               loadBalancingEnabled={loadBalancingEnabled}
+              modelRefreshIntervalInput={modelRefreshIntervalInput}
+              modelRefreshIntervalIssue={modelRefreshIntervalIssue}
               allowOriginalModelNamesForAliases={allowOriginalModelNamesForAliases}
```

ESLint

[error] Replace `allowOriginalModelNamesForAliases` with `⏎················allowOriginalModelNamesForAliases⏎··············`

(prettier/prettier)